src/moveFile.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const moveFile = (from, to) => {
+  let newTo = to;
+
+  fs.stat(to, (err, stats) => {
+    if (err) {
+      throw err;
+    }
+
+    if (stats.isDirectory()) {
+      const arr = from.split(path.sep);
+
+      newTo += path.sep + arr[arr.length - 1];
+    }
+  });
+
+  fs.rename(from, newTo, (err) => {
```

Comment:
Maybe you should choose some better methods to move files. Now it is not obvious what you are trying to do here.